dotnetv3/Keyspaces/Scenarios/KeyspacesBasics.cs

```diff
@@ -0,0 +1,230 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+namespace KeyspacesBasics;
+
+public class KeyspacesBasics
+{
+    private static ILogger logger = null!;
+
+    static async Task Main(string[] args)
+    {
+        // Set up dependency injection for the Amazon service.
+        using var host = Host.CreateDefaultBuilder(args)
+            .ConfigureLogging(logging =>
+                logging.AddFilter("System", LogLevel.Debug)
+                    .AddFilter<DebugLoggerProvider>("Microsoft", LogLevel.Information)
+                    .AddFilter<ConsoleLoggerProvider>("Microsoft", LogLevel.Trace))
+            .ConfigureServices((_, services) =>
+            services.AddAWSService<IAmazonKeyspaces>()
+            .AddTransient<KeyspacesWrapper>()
+            .AddTransient<CassandraWrapper>()
+            )
+            .Build();
+
+        logger = LoggerFactory.Create(builder => { builder.AddConsole(); })
+            .CreateLogger<KeyspacesBasics>();
+
+        var configuration = new ConfigurationBuilder()
+            .SetBasePath(Directory.GetCurrentDirectory())
+            .AddJsonFile("settings.json") // Load test settings from .json file.
+            .AddJsonFile("settings.local.json",
+                true) // Optionally load local settings.
+            .Build();
+
+        var keyspacesClient = host.Services.GetRequiredService<IAmazonKeyspaces>();
```

Comment:
You don't need to create the client here, you can let the DI do that. You just need to get the wrapper using GetRequiredService. See Route53 for an example, if that's helpful.

Reply:
Changed. Thanks for pointing that out.